guides/common/modules/proc_creating-a-remote-source-for-a-custom-file-type-repository.adoc

```diff
@@ -0,0 +1,84 @@
+[id="Creating_a_Remote_Source_for_a_Custom_File_Type_Repository_{context}"]
+= Creating a Remote Source for a {customfiletypetitle} Repository
+
+You can create a {customfiletype} repository source from a directory of files that is external to {ProjectServer} using *Pulp Manifest*.
+You can then synchronize the files into {ProjectServer} over HTTP or HTTPS.
+When you add files to a file type repository, you can work with the files as with any other repository.
+
+Use this procedure to configure a repository in a directory on a remote server.
+To create a file type repository in a directory on the base system where {ProjectServer} is installed, see xref:Creating_a_Local_Source_for_a_Custom_File_Type_Repository_{context}[].
+
+.Prerequisites
+* You have a server running {EL} 7 or 8 registered to your {Project} or the Red{nbsp}Hat CDN.
+ifdef::satellite[]
+* Your server has an entitlement to the {RHELServer} and {ProjectName} Utils repositories.
+endif::[]
+* You have installed an HTTP server.
+ifndef::orcharhino[]
+For more information about configuring a web server, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/deploying_different_types_of_servers/setting-apache-http-server_deploying-different-types-of-servers[Setting up the Apache HTTP web server] in {RHEL}{nbsp}8 _Deploying different types of servers_ or https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/system_administrators_guide/ch-web_servers#s1-The_Apache_HTTP_Server[The Apache HTTP Server] in the {RHEL}{nbsp}7 _System Administrator's Guide_.
+endif::[]
+
+.Procedure
+ifdef::satellite[]
+. On your server, ensure that the right repositories are enabled.
+** On {EL} 8:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# subscription-manager repos --enable={RepoRHEL8BaseOS} \
+--enable={RepoRHEL8AppStream} \
+--enable={RepoRHEL8ServerSatelliteUtils}
+----
+** On {EL} 7:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# subscription-manager repos --enable={RepoRHEL7Server} \
+--enable={RepoRHEL7ServerSatelliteUtils}
+----
+endif::[]
+. Install the Pulp Manifest package:
+** On {EL} 8:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# dnf install python3-pulp_manifest
+----
+** On {EL} 7:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# yum install tfm-pulpcore-python3-pulp_manifest
```

Comment:
IMHO this package is not yet available: http://yum.theforeman.org/pulpcore/3.18/el7/source/ Therefore I am unsure about the package name.

I've opened an issue against pulpcore-packaging: https://github.com/theforeman/pulpcore-packaging/issues/598

Reply:
According to theforeman/pulpcore-packaging/pull/596, these package names should be okay now.